src/app/modules/auth-private/sign-up/sign-up.component.ts

```diff
@@ -236,4 +282,58 @@ export class SignUpComponent implements OnInit {
     }
   }
 
+  onInvitationSignup() {
+    this.route.queryParams
+    .subscribe(async (params) =>  {
+      if (!isObjectEmpty(params)) {
+        const {email, first_name, last_name, consultation_id, invitation_token} = params;
+        const isActiveConsultation = await this.checkActiveConsultation(consultation_id);
+        if (isActiveConsultation) {
+          this.consultationId = consultation_id;
+          this.signupObject.firstName = first_name;
+          this.signupObject.lastName = last_name;
+          this.signupObject.email = email;
+          this.invitationToken = invitation_token;
+        }
+      }
+    });
+  }
+
+ checkActiveConsultation(consultation_id) {
```

Comment:
@Nishanthh This can now be removed, right?

Reply:
@avikt yes bro